### DISCUSSION TITLE
How can I save cycling data?

### DISCUSSION THREAD

**Discussion by naltang1** [pybamm-team/PyBaMM#2797] [Q&A]
I have a difficulty in saving the cycling data that are plotted in example of "Simulating long experiments"
When I save the data using "sol.save_data("sol_data.csv", variables=None, to_format="csv")" by follwing the Tutorial 6, they save the data of  "Current [A],Terminal voltage [V],Time [h],Throughput capacity [A.h],Loss of lithium inventory [%],Negative electrode capacity [A.h],Positive electrode capacity [A.h]". 
These are all the data that they can store, but I need to save the data of cycling data such as Cycle number, Capacity, Loss of lithium inventory, Loss of capacity to SEI, etc, that are plotted in example of "Simulating long experiments".
Please let me know how to save the data that I want. Thanks in advance

,

**Comment by valentinsulzer** [comment]
Access `solution.summary_variables`, then you have to save the data yourself, e.g. using `pandas.to_csv` (there is no helper function currently)

**Reply by naltang1** [reply]
It works. Thank you very much.
